lambda/src/main/resources/db/migration/V38__create_consignmentType_column_and_data.sql

```diff
@@ -0,0 +1,8 @@
+-- Alter the tabel and add the column 
```

Comment:
Spelling: "Alter the **table** ... "

Reply:
Corrected typo and removed data insert as discussed